src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -497,4 +497,7 @@
         Items="@(CorrelationFile)"
         Overwrite="true" />
   </Target>
+
+  <!-- Setup Dumpling service to collect and upload any crash dumps -->
+  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" />
```

Comment:
Did you try the Helix workflow? The way you have it I believe it won't work, because if I remember correctly, CloudTest.targets only gets imported in post.builds which doesn't get imported by every test project, which will cause your target to never run. If in Helix you want this to work on a per project basis, you might want to instead have a way of setting `EnableDumpling` on Helix builds, or if you want it to run on a full build basis, then you might want to write a different target that runs on Helix builds that collects all dumps.